include/gsl/span

```diff
@@ -463,18 +464,17 @@ public:
 
     constexpr span<element_type, dynamic_extent> last(index_type count) const
     {
-        Expects(count >= 0 && count <= size());
-        return {data() + (size() - count), count};
+        Expects(count >= 0 && size() - count >= 0);
```

Comment:
Very minor nit.  Any particular reason you're still using Expects here?  It isn't needed as `make_span` covers it.  In your change for line 474 in `sub_span` did drop Expects.

Reply:
You are right, the checks are not needed. Will update the PR.